cmd/ebrelayer/main.go

```diff
@@ -211,6 +210,7 @@ func RunInitRelayerCmd(cmd *cobra.Command, args []string) error {
 
 	logConfig := zap.NewDevelopmentConfig()
 	logConfig.Sampling = nil
+	logConfig.Encoding = "json"
```

Comment:
Are you hardcoding log output as Json and not using a flag, is that something we always want to be JSON?

Reply:
Yes, right now it's hardcoded.  We should probably add a feature to let users select this, but I thought we should get it working with json first.